src/track/cueinfoimporter.cpp

```diff
@@ -0,0 +1,79 @@
+#include "track/cueinfoimporter.h"
+
+namespace mixxx {
+
+CueInfoImporter::CueInfoImporter() {
+}
+
+CueInfoImporter::CueInfoImporter(const QList<CueInfo>& cueInfos)
+        : m_cueInfos(cueInfos) {
+}
+
+double CueInfoImporter::getTimingOffsetMillis(
+        const QString& filePath,
+        const audio::SignalInfo& signalInfo) const {
+    Q_UNUSED(filePath);
+    Q_UNUSED(signalInfo);
+    return 0;
+};
+
+void CueInfoImporter::append(const CueInfo& cueInfo) {
+    m_cueInfos.append(cueInfo);
+}
+
+void CueInfoImporter::append(const QList<CueInfo>& cueInfos) {
+    m_cueInfos.append(cueInfos);
+}
+
+int CueInfoImporter::size() const {
+    return m_cueInfos.size();
+}
+
+bool CueInfoImporter::isEmpty() const {
+    return m_cueInfos.isEmpty();
+}
+
+QList<CueInfo> CueInfoImporter::getCueInfosWithCorrectTiming(
```

Comment:
Use a verb other than "get", e.g. "importCueInfosWithCorrectTiming". This is an "importer" because it's purpose is to "import".